src/Social/Services/Feeds.php

```diff
@@ -64,7 +65,7 @@ public static function getByChannel(Channels $channel): Simple
      * @param string $distribution
      * @return UserMessages
      */
-    public static function create(UserInterface $user, string $verb, array $message = [], array $object = null, string $distribution = 'profile'): Messages
+    public static function create(UserInterface $user, string $verb, array $message = [], MessageableInterface $object = null): Messages
```

Comment:
@arfenis if we accept MessageableInterface to null, than it should be ?MessageableInterface